From 3663d74d83601e1781f6806ed9b58a186edb3386 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:49 -0400
Subject: [ethernet] i40e: remove usless return statements

Message-id: <1409663591-32531-67-git-send-email-sassmann@redhat.com>
Patchwork-id: 90545
O-Subject: [RHEL7.1 PATCH 066/148] i40e, igb, ixgbe: remove usless return statements
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Stephen Hemminger <stephen@networkplumber.org>

Remove cases where useless bare return is left at end of function.

Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 41457f64da53112996d2ac607fbb2681e71a9e97)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 6f6bd3f..8dbaa77 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -693,7 +693,6 @@ static void i40evf_del_vlan(struct i40evf_adapter *adapter, u16 vlan)
   f->remove = true;
   adapter->aq_required |= I40EVF_FLAG_AQ_DEL_VLAN_FILTER;
  }
- return;
 }
 
 /**
@@ -1232,8 +1231,6 @@ void i40evf_reset_interrupt_capability(struct i40evf_adapter *adapter)
  pci_disable_msix(adapter->pdev);
  kfree(adapter->msix_entries);
  adapter->msix_entries = NULL;
-
- return;
 }
 
 /**
@@ -2158,7 +2155,6 @@ err:
   return; /* do not reschedule */
  }
  schedule_delayed_work(&adapter->init_task, HZ * 3);
- return;
 }
 
 /**
-- 
1.7.1